lib/src/one_hot.dart

```diff
@@ -29,22 +29,28 @@ class BinaryToOneHot extends Module {
 class OneHotToBinary extends Module {
   /// The [binary] decoded result.
   Logic get binary => output('binary');
+  Logic get error => output('error');
 
   /// Constructs a [Module] which decodes a one-hot or thermometer-encoded
   /// number [onehot] into a 2s complement number [binary] by encoding
   /// the position of the '1'
   OneHotToBinary(Logic onehot) {
     onehot = addInput('onehot', onehot, width: onehot.width);
     addOutput('binary', width: log2Ceil(onehot.width + 1));
+    addOutput('error');
```

Comment:
Ideally we could make the error signal optional (see `fifo.dart` for an example), then `error` would be a `Logic?` depending on whether `generateError` was true or false.  If you want to punt this to another issue/PR/someone else that's fine too

Reply:
done